test/src/updater/nsisUpdaterTest.ts

```diff
@@ -46,6 +46,17 @@ test("file url generic", async () => {
   await validateDownload(updater)
 })
 
+test.ifEnv(process.env.KEYGEN_TOKEN)("file url keygen", async () => {
+  const updater = await createNsisUpdater()
+  updater.addAuthHeader(`Bearer ${process.env.KEYGEN_TOKEN}`)
```

Comment:
Hey @ezekg! Quick note for documentation: The end-user will need to know this detail of adding the "Bearer " prefix on their own. I think the function's name makes it self-explanatory that it's the `Authorization` header, but might still be worth noting in the docs.